app/assets/javascripts/feedback_form.js

```diff
@@ -44,17 +44,17 @@ Blacklight.onLoad(function(){
           $thisform.on('submit', function(e){
             e.preventDefault();
             var valuesToSubmit = $(this).serialize();
-            $.ajax({
-              url: form.action,
-              data: valuesToSubmit,
-              type: 'post'
-            }).success(function(response){
-              if (isSuccess(response)){
-                $($el).collapse('hide');
-                $($form)[0].reset();
-              }
-              renderFlashMessages(response);
-            });
+            fetch(form.action, {
+              method: 'POST',
+              headers: { "Content-Type": "application/x-www-form-urlencoded" },
+              body: valuesToSubmit
+            }).then((resp) => resp.json())
```

Comment:
I think if you make `body` a `URLSearchParams` it'll automatically send the content-type as `x-www-form-urlencoded`. That might let you skip using jQuery's `serialize()` too; I think it would just be `new URLSearchParams(new FormData($thisform[0]))`